webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableFormats.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+export const hostsCountFormat = (value, { rowData: { name } }) => (
+  <a href={`hosts?search=model+%3D+"${encodeURI(name)}"`}>{value}</a>
+);
+
+export const actionsFormat = (value, { rowData: { name, id } }) =>
+  (value.includes('delete') ? (
+    <span className="btn btn-sm btn-default">
+      <a
+        data-confirm={__(`Delete ${name}`)}
+        data-id={`aid_models_${id}-${encodeURI(name)}`}
+        rel="nofollow"
+        data-method="delete"
+        href={`/models/${id}-${encodeURI(name)}`}>
+        Delete
+      </a>
+    </span>
+  ) : null);
+
+export const nameFormat = (value, { rowData: { actions, id, name } }) =>
+  (actions.includes('edit') ? (
+    <a
+      href={`models/${id}-${encodeURI(name)}/edit`}
+      data-id={`aid_models_${id}-${encodeURI(name)}_edit`}>
+      {value}
+    </a>
+  ) : (
+    <a href="#" className="disabled" disabled="disabled" onClick={() => false}>
+      {value}
+    </a>
+  ));
+
+const orderChars = {
+  ASC: '▲',
```

Comment:
looking at PF documentation at https://www.patternfly.org/pattern-library/

they have the following html for table headers:
```html
<th
  class="sorting_asc"
  tabindex="0"
  aria-controls="patternTable"
  rowspan="1"
  colspan="1"
  aria-sort="ascending"
  aria-label="Pattern: activate to sort column descending"
>
  Pattern
</th>
```